xml/obs_glossary.xml

```diff
@@ -250,25 +252,17 @@
   <glossterm>Container</glossterm>
   <glossdef>
    <para>
-    A container is an image file that contains a deployable
-    version of software and metadata. Included with the main software are
-    usually also dependencies of it, such as additional libraries. Containers
-    can also include, for example, configuration files and documentation.
-    However, unlike images, containers do not include an operating system.
+    An image file that contains a deployable version of software and
+    metadata. Dependencies of the main software are also included, such as
+    additional libraries.
    </para>
    <para>
-    Containers cannot be installed. Instead, they are copied as-is onto the
-    target system (<emphasis>deployed</emphasis>). They can then usually be
-    run directly in that state.
+    Unlike operating system images, containers do not include an operating
+    system. Unlike binary packages, containers are deployed and not installed.
+    Formats of containers include &appimg;, &docker;, &snap;, and &flatpak;.
    </para>
    <para>
-    You can build containers on &obsa;. &obsa; supports
-    building multiple types of container. Container formats include
-    &appimg;, &docker;, and &snap;. <!-- &flatpak; ... later. -->
-   </para>
-   <para>
-    Alternatively, &obsa; can also build binary packages and operating system
-    images.
+    You can build containers on &obsa;.
```

Comment:
This is interesting and I never thought about to include it. 😉

However, I'm not sure if we should start to add supported features into a definition. Shouldn't be a definition something "pure" and independent of any features?